### PR TITLE
Updated guide for Neo4j Browser and created guide for Neo4j Desktop

### PR DESCRIPTION
**Pull request by JMHReif**
Would you guys review? Let me know in comments if there's anything that needs changed. This is just for adding the Neo4j for desktop guide and updating the Neo4j browser guide. I also removed old image files that were outdated (cleanup), plus added the link on the left-side menu for the new guide. Let me know what you think!

Thanks,
Jennifer

### CODE REVIEW

File: developer-resources [hunk 1]
```diff
@@ -1 +0,0 @@
-developer-resources
```
Comment (jexp) [inline developer-resources:1]: why did you delete the directories/symlinks? developer-resources and deploy?

File: get-started/index.html [hunk 2]
```diff
@@ -952,7 +953,7 @@ <h5>Data Modeling</h5>
             <dl class="accordion">
                <dd class="accordion-navigation inactive">
                   <a href="/developer/working-with-data/">
-                     <h5>Working with Data</h5>
+                     <h5>Data Import & Visualization</h5>
```
Comment (jexp) [inline get-started/index.html:956]: I'd love to discuss this topic, going forward, for now this covers the content below.

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc [hunk 3]
```diff
@@ -8,45 +8,46 @@
...
 == Installing and Starting Neo4j Browser
 
-First, http://neo4j.org/download[download] Neo4j. 
-Use the provided installer, start the Neo4j Desktop App and start the database after (optionally) selecting a location for your database files.
-
-Alternatively unzip the downloaded file, and start Neo4j by following the instructions in the readme (`bin/neo4j start`).
-
+If you haven't already, http://neo4j.org/download[download^] Neo4j. 
+Using the provided instructions (shown when downloading), follow Step 1 to install and start the Neo4j Desktop application.
+Instructions can also be found in the readme (`bin/neo4j start`) after unzipping the downloaded file.
```
Comment (jexp) [inline get-started/guide-neo4j-browser/guide-neo4j-browser.adoc]: That's a separate download. Perhaps mention that "Alternatively, after downloading the zipped server-only package". We should probably recommend running `bin/neo4j console` these days and also show the windows equivalent `\bin\neo4j.bat console`  (in a terminal)

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc [hunk 4]
```diff
@@ -8,45 +8,46 @@
...
+Step 2 on the download page instructions shows you how to set up your first project and database and how to open the Neo4j in the browser.
```
Comment (jexp) [inline get-started/guide-neo4j-browser/guide-neo4j-browser.adoc]: > How to open the Neo4j-Browser.

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc [hunk 5]
```diff
@@ -8,45 +8,46 @@
...
+If you forget it, you have to remove the credentials file in `data/dbms/authorization`.
 ****
 
-Open the URL http://localhost:7474 to start working with Neo4j and our query language, link:/developer/cypher[Cypher].
+Just to review, so far, we have shown two ways to interact with Neo4j.
+
+1) From Neo4j Desktop, you can open Neo4j Browser. This is done my clicking `<your project name>` in the Desktop screen, then click `Manage` on your database.
 
-// tag::guide[]
+2) You can also access this from a regular browser window by typing http://localhost:7474 and signing in with Username: neo4j, Password: <your database password (set up in Desktop)>.
```
Comment (jexp) [inline get-started/guide-neo4j-browser/guide-neo4j-browser.adoc]: you might want to use `1.` if you want to have a numbered list?

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc [hunk 6]
```diff
@@ -55,11 +56,60 @@ You can provide your name and send us a message.
...
+
+== Built-in Guides
+
+If you want to learn more about Neo4j, Cypher, or the Neo4j browser before diving in, click on any of the helpful links shown after startup (commands also shown below).
+These are quick guides that introduce the different concepts.
+You can find more helpful links in the left sidebar of Neo4j in the "Documents" tab (icon:book[]).
+
+[%autowidth.spread]
+|===
+|*Title* |*Description* |*Command*
```
Comment (jexp) [inline get-started/guide-neo4j-browser/guide-neo4j-browser.adoc]: options=header in the table
you can also format the third column as monospace, I think with cols="a,a,m" 
https://asciidoctor.org/docs/user-manual/#tables

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc [hunk 7]
```diff
@@ -55,11 +56,60 @@ You can provide your name and send us a message.
...
+|Intro |A guided tour of Neo4j browser |`:play intro`
+|Concepts |Graph database basics |`:play concepts`
+|Cypher |Neo4j's graph query language introduction |`:play cypher`
+|The Movie Graph |A mini graph model of connections between actors and movies |`:play movie graph`
+|The Northwind Database |A classic use case of RDBMS to graph with import instructions and queries |`:play northwind graph`
+|Custom Guides |Starting with Neo4j 2.3, you can use _:play <url>_ to play a custom guide |`:play
```
Comment (jexp) [inline get-started/guide-neo4j-browser/guide-neo4j-browser.adoc]: we can remove the version reference here, but we should point to: https://neo4j.com/developer/guide-create-neo4j-browser-guide/

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc [hunk 8]
```diff
@@ -55,11 +56,60 @@ You can provide your name and send us a message.
...
+
+http://guides.neo4j.com/graphgist/restaurant_recommendation.html`
+|===
+
+All you have to do is enter any of the above commands in the browser command line (at the top of the browser) and follow the accompanying guides to walk through the training or scenarios.
```
Comment (jexp) [inline get-started/guide-neo4j-browser/guide-neo4j-browser.adoc]: mention that the guides are slide-shows?

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc [hunk 9]
```diff
@@ -55,11 +56,60 @@ You can provide your name and send us a message.
...
+If you click on any label or relationship *above* the graph visualization, you can then choose its styling in the area below the graph.
+Colors, sizes, and captions are selectable from there.
+
+To see this for yourself, you can click on the `Person` label above the graph and change the color, size, and captions of all nodes labeled with `Person`.
+The first image below shows changes to nodes labeled `Person`. The second image shows changes to relationships labeled `ACTED_IN`.
+
+image::{img}style_person_node.png[]
 
-Just log in with your GitHub, Twitter or Google account and synchronize your workplace to the cloud.
+image::{img}style_actedin_relationship.png[]
```
Comment (jexp) [inline get-started/guide-neo4j-browser/guide-neo4j-browser.adoc:117]: perhaps mention full screen and that you can zoom there and that it keeps the zoom level even if you go back?

Comment (jexp) [reply get-started/guide-neo4j-browser/guide-neo4j-browser.adoc:117]: mention the "Auto connect nodes" in the settings for graph-viz
i.e. that it auto-fetches relationships between all nodes and if you don't want that, e.g. b/c you return select relationships you can disable it there.

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc [hunk 10]
```diff
@@ -97,174 +147,111 @@ m|:help keys
...
-|===
-
-Import our sample movie graph by entering `:play movie graph`.
-
-On the second slide click the large Cypher +CREATE ...+ statement, then hit the triangular `Run` button.
+== Query and Command-Line Tips
 
-After a few seconds the data is imported, and you'll see a subset of the movie data rendered as a graph.
+You can remove all accumulated output frames with `:clear`. The 'X' button at the top right of each pane removes that frame and aborts a (long-)running statement.
+The maximum number of frames that are kept is configurable in the Browser Settings from the left-side menu.
```
Comment (jexp) [inline get-started/guide-neo4j-browser/guide-neo4j-browser.adoc]: mention pinning of frames (e.g. guides) so they stay on top?
perhaps best in the guides section.

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc [hunk 11]
```diff
@@ -97,174 +147,111 @@ m|:help keys
...
-To execute, hit the triangular play button.
-
-Query results are rendered either as visual graph, tabular or ascii-table result.
-You can switch between those with the icons on the left side of the result frame.
+If you want to review a past query, you can find the result pane and click the query above the graph visualisation to pull it back into the editor.
+The keyboard shortcuts listed above will help you work efficiently within the editor area.
 
-Remove all accumulated output frames with `:clear`, the cross removes a single frame and aborts a (long-)running statement.
-The maximum number of frames that are kept is configurable.
+You can also write and edit multi-line queries by switching to multi-line editing mode with kbd:[Shift+Enter], then kbd:[Enter] will create newlines.
```
Comment (jexp) [inline get-started/guide-neo4j-browser/guide-neo4j-browser.adoc]: you then need to run Ctrl+Enter or Cmd+Enter to run your queries.

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc [hunk 12]
```diff
@@ -97,174 +147,111 @@ m|:help keys
...
-You can click the small query above the graph visualisation to pull it back into the editor.
+kbd:[Ctrl+Up] and kbd:[Ctrl+Down] (Mac users, use the kbd:[Cmd] key) allows you to navigate command history, and you can access all command history with `:history`.
+The command history is persisted across browser restarts.
 
-Use the keyboard shortcuts listed above to work efficiently with the editor area.
+You can switch between `Graph`, `Table`, `Text`, and `Code` views to see the results in various formats by clicking the icons on the left of each pane.
 
-Navigate input history with kbd:[Ctrl+Up] and kbd:[Ctrl+Down], access all of it via `:history`. 
-The history will be persisted across browser restarts.
+****
```
Comment (jexp) [inline get-started/guide-neo4j-browser/guide-neo4j-browser.adoc]: I don't think we need both the `****` box and the `NOTE` admonition?

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc [hunk 13]
```diff
@@ -97,174 +147,111 @@ m|:help keys
...
+You can also turn previously dragged nodes loose again.
 
-If you enter fullscreen mode of a graph visualization you can zoom in and out.
-After a node is clicked it gets a halo, in which you can expand and remove nodes from the visualization. 
-You can also turn previously dragged nodes lose again.
+== Graph Metadata
 
-== Meta Graph
+In the left-side menu, the top icon is the database section (icon:database[]), where you can find the *currently used* node labels, relationship types, and property keys.
+Clicking on any one of those options runs a quick query to show you a sample of the graph with those elements.
```
Comment (jexp) [inline get-started/guide-neo4j-browser/guide-neo4j-browser.adoc:195]: mention the additional canned/frequently used queries that also include a visual "schema" query. (i.e. `call db.schema()`

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc [hunk 14]
```diff
@@ -97,174 +147,111 @@ m|:help keys
...
-This executes the statement `MATCH (n) RETURN n limit 100` which fetches some nodes. 
+You can save your own queries as favorites by "starring" them.
+Just populate the Browser command line with the query you want to favorite, then click the Favorites (icon:star-o[]) icon to the right of the command line.
+This will add the query to your Favorites list in the left-side menu.
+To run one of your Favorites, click on the left-side menu Favorites, choose the query, and run it.
 
-The browser helpfully also fetches and displays relationships between those nodes, *even if they were not part of your query result*.
-You can disable the latter behavior with the "Auto-Complete" switch in the bottom right corner.
-Then only relationships returned by the actual query will be shown.
+To provide a title or helpful info, you can use a comment `// comment` above your query to provide a title.
```
Comment (jexp) [inline get-started/guide-neo4j-browser/guide-neo4j-browser.adoc]: which is used by the Favorites menu

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc [hunk 15]
```diff
@@ -97,174 +147,111 @@ m|:help keys
...
-You can save your own queries as favorites by "starring" them. 
-Use a comment `// comment` above your query to provide a title.
-Use folders to organize the favorites you can rearrange them by dragging and delete if they are no longer useful.
+Creating folders can help organize your favorites, and you can rearrange them by dragging or delete them if they are no longer useful.
 
 [NOTE]
 Your favorites, settings, and styles can be saved to the cloud with Neo4j Browser Sync.
 If you clear your browser cache, your local favorites will be removed.
 
-=== Advanced Styling
```
Comment (jexp) [inline get-started/guide-neo4j-browser/guide-neo4j-browser.adoc]: it would be cool if you recorded a screencast at some point :)

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc [hunk 16]
```diff
@@ -97,174 +147,111 @@ m|:help keys
...
+=== Advanced Styling
 
-For more advanced styling you can bring up the style-viewer with `:style`, download the graph-style-sheet (GRASS), edit it offline and drag it back onto the drag-area of the viewer.
+For more advanced styling, you can bring up the style-viewer with `:style` and copy/paste the graph-style-sheet (GRASS) that is returned.
+You can edit this stylesheet offline, save the file as a .grass file, and drag it back onto the drag-area of the viewer.
 
 [NOTE]
-You can reset to the default styles with `:style reset`.
-Alternatively by clicking the "fire extinguisher" icon in the popup from `:style`.
+You can reset to the default styles with `:style reset` or by clicking the "fire extinguisher" icon in the popup from `:style`.
```
Comment (jexp) [inline get-started/guide-neo4j-browser/guide-neo4j-browser.adoc]: the fire extinguisher doesn't exist anymore as does the popup :(

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc [hunk 17]
```diff
@@ -97,174 +147,111 @@ m|:help keys
...
+* You can retrieve the current configuration with `:config`.
+* Individual settings are configured with the following defaults:
 ** `:config maxNeighbours:100` - maxiumum number of neighbours for a node
 ** `:config maxRows:100` - maximum number of rows for the tabular result
 
 == Executing REST requests
 
-You can also execute REST requests with the Neo4j Browser, the command-syntax is +
-`:COMMAND /a/path {"some":"data"}`.
+You can also execute REST requests with the Neo4j in the browser. The command syntax is `:COMMAND /a/path {"some":"data"}`.
```
Comment (jexp) [inline get-started/guide-neo4j-browser/guide-neo4j-browser.adoc]: did you check that this still works? it was broken at some point in the new browser after the rewrite.

Comment (jexp) [reply get-started/guide-neo4j-browser/guide-neo4j-browser.adoc]: Not sure if we need to keep the whole REST section here, as this is quite specific stuff, and probably too much for an "Intro" ?

File: get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc [hunk 18]
```diff
@@ -0,0 +1,147 @@
...
+
+toc::[]
+
+//:img: .
+
+== Installing and Starting Neo4j Desktop
+
+If you haven't already, http://neo4j.org/download[download^] Neo4j. 
+Using the provided instructions (shown when downloading), follow Step 1 to install and start the Neo4j Desktop application.
+Instructions can also be found in the readme (`bin/neo4j start`) after unzipping the downloaded file.
```
Comment (jexp) [inline get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc]: remove this line

File: get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc [hunk 19]
```diff
@@ -0,0 +1,147 @@
...
+Step 2 on the download page instructions shows you how to set up your first project and database and how to open the Neo4j in the browser, but we want to cover this step in more detail here.
```
Comment (jexp) [inline get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc]: shouldn't we do this here? as that download pages is most likely gone already?

File: get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc [hunk 20]
```diff
@@ -0,0 +1,147 @@
...
+:experimental:
+:neo4j-version: 3.3.4
+
+.Goals
+[abstract]
+This article demonstrates how to use Neo4j for desktop for managing instances of Neo4j locally for development.
+
+.Prerequisites
+[abstract]
+Please have link:/download[Neo4j^] downloaded and installed.
```
Comment (jexp) [inline get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc]: not really need it as it's covered below

File: get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc [hunk 21]
```diff
@@ -0,0 +1,147 @@
...
+At the top, the folder with a bookmark (image:{img}projects_icon.png[width=20]) shows the list of projects you have created.
+If this is the first time using Neo4j for desktop, the default `My Project` will be the only one.
+You can either create a new project to work with your use case or create your database within the default project, but this guide will use the default `My Project`.
+
+image::{img}desktop_mainscreen.png[]
+
+=== Creating and Deleting a Project
+
+To create a new project, simply click `+ New` at the right of the `Projects` menu.
+The app will add a row in the menu for a project named `Project`. You can change the name by clicking on the pencil icon that appears when you hover on the project name in the right pane.
```
Comment (jexp) [inline get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc]: sentence per line :)

File: get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc [hunk 22]
```diff
@@ -0,0 +1,147 @@
...
+
+The bottom icon (image:{img}neo4j_icon.png[width=20]) is information about Neo4j itself and includes links to the main company website, as well as release and license information.
+
+== Creating a graph database
+
+From the Projects icon on the left-side menu, choose (or create) a project.
+A panel will display on the right to show the databases in your project and the plugins available or installed.
+
+To create a database, click in the dotted square that says `New Graph`.
+Then click the kbd:[Create a Local Graph] button.
```
Comment (jexp) [inline get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc]: how do these render? as `kbd:` is originally a macro for rendering keyboard codes.

File: get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc [hunk 23]
```diff
@@ -0,0 +1,147 @@
...
+Next, you will type in a name for your database and set a password and choose a version (default is latest version), then click kbd:[Create].
+It may take a few minutes to download the version and actually create the database.
```
Comment (jexp) [inline get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc]: mention that it shows with a little icon which versions have been downloaded already

File: get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc [hunk 24]
```diff
@@ -0,0 +1,147 @@
...
+image::{img}db_section.png[]
+
+== Managing a database
+
+When you click the kbd:[Manage] button, the resulting pane opens up several functionalities.
+
+image::{img}manage_db.png[]
+
+In the top section is your database name with buttons underneath it for Start, Stop, and Restart of the database.
+The buttons next to those allow you to open the folder location of the database (for modifying data files and configuration) and to open the Neo4j Browser to interact with data in the database.
```
Comment (jexp) [inline get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc]: also to access e.g. the import folder for placing CSV files
with the drop-down on the right side.

File: get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc [hunk 25]
```diff
@@ -0,0 +1,147 @@
...
+
+In the bottom section are several tabs for information and settings.
+The `Details` tab shows the version and the status of your database.
+When the database is running, it will also show port numbers and addresses needed to interact with the database (screenshot below).
+
+image::{img}db_details_tab.png[]
+
+The `Logs` tab will show all of the streaming log output from the database.
+
+`Terminal` tab allows you to interact with the database at the command line and potentially retrieve/send data via any end points set up.
```
Comment (jexp) [inline get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc]: e.g. to run `cypher-shell` or look at other logs

File: get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc [hunk 26]
```diff
@@ -0,0 +1,147 @@
...
+
+The `Settings` tab displays the configuration values for the database.
+These can be changed, if needed.
```
Comment (jexp) [inline get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc]: then the Desktop offers to restart the database after you `Apply` your changes.

File: get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc [hunk 27]
```diff
@@ -0,0 +1,147 @@
...
+
+****
+[NOTE]
+You can also search the settings in this tab by using kbd:[Ctrl]+F / kbd:[Cmd]+F (Mac).
```
Comment (jexp) [inline get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc]: I think the `+F` has to be inside the `[Cmd+F]`

File: get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc [hunk 28]
```diff
@@ -0,0 +1,147 @@
...
+****
+
+On the `Plugins` tab, you can see what plugins are available (or you have installed) to use with Neo4j.
+Currently, Neo4j has plugins for APOC, GraphQL, and Graph Algorithms.
```
Comment (jexp) [inline get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc]: Neo4j Desktop has

File: get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc [hunk 29]
```diff
@@ -0,0 +1,147 @@
...
+The `Upgrade` tab shows the list of all Neo4j versions, as well as the version this instance is currently running.
+To change the version, choose one from the list and click kbd:[Upgrade to this version] in the right pane.
+
+The last tab is `Administration`.
+This tab just allows you to set a new password for your database.
+
+== Feedback & Questions
+
+If you have feedback or questions on how to use Neo4j for desktop, feel free to reach out to us.
+You can submit messages to us through Intercom and tag as related to `desktop`.
```
Comment (jexp) [inline get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc]: should we explain where they find intercom? @ryguyrg ? and how tagging works?

File: get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc [hunk 30]
```diff
@@ -0,0 +1,147 @@
...
+
+== Next Steps
+
+Now that we covered the basics of Neo4j for desktop, you can start working with data using our query language, link:/developer/cypher[Cypher^].
```
Comment (jexp) [inline get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc]: Neo4j Desktop

**Review by jexp — CHANGES_REQUESTED** [review]
Please have a look at my comments.
Great work well done.

File: developer-resources (hunk 1 above)
Comment (JMHReif) [reply developer-resources:1]: Those were sym directories that I think I accidentally added in my last release. The developer-resources contained a copy of some really old changes I made when I first started at Neo. So I removed them.

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc (hunk 3 above)
Comment (JMHReif) [reply get-started/guide-neo4j-browser/guide-neo4j-browser.adoc]: Ok. That makes sense. I'd need access to a Windows machine before I can run the Windows steps, though. Why should we recommend running bin/neo4j console (i.e. what does that do)?

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc (hunk 6 above)
Comment (JMHReif) [reply get-started/guide-neo4j-browser/guide-neo4j-browser.adoc]: Sorry, I didn't follow this one. Why do I need options=header and how do I put that in the table? Also, is monospace the same as the `syntax`?

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc (hunk 9 above)
Comment (JMHReif) [reply get-started/guide-neo4j-browser/guide-neo4j-browser.adoc:117]: Full screen for the Browser window or full screen for the graph pane? Just want to make sure I put it in the right place. :)
Is that the "Connect result nodes" checkbox at the bottom? What are the cases where you would want to auto-fetch relationships versus not? Not sure I understand this functionality...

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc (hunk 10 above)
Comment (JMHReif) [reply get-started/guide-neo4j-browser/guide-neo4j-browser.adoc]: I actually just discovered this functionality yesterday! 😆  I'll add it.

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc (hunk 13 above)
Comment (JMHReif) [reply get-started/guide-neo4j-browser/guide-neo4j-browser.adoc:195]: Is there a list of those somewhere? How do they know what to run?

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc (hunk 14 above)
Comment (JMHReif) [reply get-started/guide-neo4j-browser/guide-neo4j-browser.adoc]: I didn't know this! Adding now.

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc (hunk 15 above)
Comment (JMHReif) [reply get-started/guide-neo4j-browser/guide-neo4j-browser.adoc]: Agree! Thought I'd get this started first, then add a bunch of screencasts to several pages later. :)

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc (hunk 16 above)
Comment (JMHReif) [reply get-started/guide-neo4j-browser/guide-neo4j-browser.adoc]: Fixed!

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc (hunk 17 above)
Comment (JMHReif) [reply get-started/guide-neo4j-browser/guide-neo4j-browser.adoc]: Oops, I forgot to test this section! Just tested now, and they do still work. I agree that most beginners probably wouldn't use it, but I'm afraid we'd get questions about it if we didn't include it. I didn't realize you could do this before I read this section. I'm tempted to leave it there. If someone's curious, at least we have the info somewhere. Don't you think?

File: get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc (hunk 18 above)
Comment (JMHReif) [reply get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc]: The line on "Instructions can also be found..."?

File: get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc (hunk 19 above)
Comment (JMHReif) [reply get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc]: I thought about it, but I didn't want to duplicate the instructions, especially since they could change frequently with Neo4j updates. The only step I don't walk them through is the download of Neo4j, which is the same as any other internet download and the download instructions don't even really say much (just open installer, save to apps, open the app).

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc (hunk 9 above)
Comment (jexp) [reply get-started/guide-neo4j-browser/guide-neo4j-browser.adoc:117]: full-screen graph pane.

usually you want to auto-fetch when you just explore and don't bother to return exact relationships or full paths

but if you need to examine production queries from an application you exactly want to see only what the query returned, then you can disable it.

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc (hunk 13 above)
Comment (jexp) [reply get-started/guide-neo4j-browser/guide-neo4j-browser.adoc:195]: a bunch of them are in the favorites sidebar and some other places in the UI
like the security ones, or listing indexes/constraints
all built in procedures are listed here: https://neo4j.com/docs/operations-manual/current/reference/procedures/

File: get-started/guide-neo4j-browser/guide-neo4j-browser.adoc (hunk 6 above)
Comment (jexp) [reply get-started/guide-neo4j-browser/guide-neo4j-browser.adoc]: ya, so that you have to spell it out each time, mixing content and styling when you can declare it on a per-column basis.

File: get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc (hunk 19 above)
Comment (jexp) [reply get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc]: I still think we should have a stable place that people can find this information and the download page is really transient and almost impossible to get back except for when downloading another 300MB file. WDYT @ryguyrg

File: get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc (hunk 29 above)
Comment (jexp) [reply get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc]: @ryguyrg ?
